CheckpointPlugin.cpp

```diff
@@ -119,6 +119,32 @@ void CheckpointPlugin::onLoad()
 	boolvar("cpt_mirror_loads", "If set, randomly mirror when loading checkpoints", &mirrorLoads);
 	boolvar("cpt_randomize_loads", "If set, load a random checkpoint instead of the latest", &randomizeLoads);
 
+	// Register CVars for action thresholds and enable/disable toggles
+	cvarManager->registerCvar("enable_throttle_unpause", "1", "Enable throttle to unpause", true, true, 0, true, 1, true);
+	cvarManager->registerCvar("throttle_threshold", "0.1", "Threshold for throttle to unpause", true, true, 0, true, 1, true);
+
+	cvarManager->registerCvar("enable_steer_unpause", "0", "Enable steering to unpause", true, true, 0, true, 1, true);
+	cvarManager->registerCvar("steer_threshold", "0.1", "Threshold for steering to unpause", true, true, 0, true, 1, true);
+
+	cvarManager->registerCvar("enable_pitch_unpause", "1", "Enable pitch to unpause", true, true, 0, true, 1, true);
+	cvarManager->registerCvar("pitch_threshold", "0.1", "Threshold for pitch to unpause", true, true, 0, true, 1, true);
+
+	cvarManager->registerCvar("enable_yaw_unpause", "1", "Enable yaw to unpause", true, true, 0, true, 1, true);
+	cvarManager->registerCvar("yaw_threshold", "0.1", "Threshold for yaw to unpause", true, true, 0, true, 1, true);
+
+	cvarManager->registerCvar("enable_roll_unpause", "1", "Enable roll to unpause", true, true, 0, true, 1, true);
+	cvarManager->registerCvar("roll_threshold", "0.1", "Threshold for roll to unpause", true, true, 0, true, 1, true);
+
+	cvarManager->registerCvar("enable_handbrake_unpause", "1", "Enable handbrake to unpause", true, true, 0, true, 1, true);
+	cvarManager->registerCvar("enable_jump_unpause", "1", "Enable jump to unpause", true, true, 0, true, 1, true);
+	cvarManager->registerCvar("enable_boost_activate_unpause", "1", "Enable boost activation to unpause", true, true, 0, true, 1, true);
+	cvarManager->registerCvar("enable_boost_hold_unpause", "1", "Enable boost hold to unpause", true, true, 0, true, 1, true);
+
+
+	cvarManager->registerCvar("rewind_input", "steer", "Input used for rewinding (steer, throttle, pitch, yaw, roll)");
```

Comment:
How would "throttle" be used to rewind?  Does the brake/reverse do fast-forward?  Is that under the "throttle" input using negative values for reverse?  (Have you tested it?)